target/linux/ipq40xx/files-5.4/drivers/net/phy/ar40xx.h

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright (c) 2016, The Linux Foundation. All rights reserved.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for
+ * any purpose with or without fee is hereby granted, provided that the
+ * above copyright notice and this permission notice appear in all copies.
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT
+ * OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+ #ifndef __AR40XX_H
```

Comment:
This is actually indented (like in the patch). Maybe one should fix this in a separate patch as well.

Reply:
Hm, I applied the patches and then copied ar40xx.c and ar40xx.h from the patched kernel tree, so its gotta be exactly the same as in the patch

Reply:
The space is also there in the patch, it's not your fault. Nevertheless, this looks broken to me (as #define only works at the beginning of the line?), so I wanted to ask for your opinion ...

Reply:
Yeah, space should not be here.
It most likely works as the preprocessor will take care of that anyway